pkg/handlers/search_files.go

```diff
@@ -0,0 +1,73 @@
+package handlers
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/artmoskvin/hide/pkg/model"
+	"github.com/artmoskvin/hide/pkg/project"
+	"github.com/rs/zerolog/log"
+)
+
+const (
+	queryKey = "query"
+)
+
+type SearchFilesHandler struct {
+	ProjectManager project.Manager
+}
+
+func (h SearchFilesHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	projectID, err := getProjectID(r)
+	if err != nil {
+		http.Error(w, fmt.Sprintf("Invalid project ID: %s", err), http.StatusBadRequest)
+		return
+	}
+	log.Info().Msgf("project id %s", projectID)
```

Comment:
Probably worth to reduce the level to DEBUG and add a more meaningful message

Reply:
Forgot to delete this line.